mods/cnc/languages/rules/en.ftl

```diff
@@ -559,3 +559,13 @@ actor-truck =
       Builds fast
       Unarmed
    .name = Supply Truck
+
+## ai.yaml
+cabal =
```

Comment:
Would it make sense to give these a `bot-` prefix, like actors have `actor-` above?